### DISCUSSION TITLE
How do I reduce the forwarding speed?

### DISCUSSION THREAD

**Discussion by ghost** [aahnik/tgcf#113] [Q&A] [answered]
How do I reduce the forwarding speed so that i don't get the following error. 

Sleeping for A wait of 1498 seconds is required (caused by SendMediaRequest)

I get this error after every 10 minutes and the wait time keeps increasing every time. I have one source and 4 destinations if it helps.

**Comment by aahnik** [comment, marked as answer]
hi @adamgilli , this is already answered. Please don't ask duplicate questions.

Read these:
- https://github.com/aahnik/tgcf/wiki/Flood-wait-errors-and-bans
- https://github.com/aahnik/tgcf/issues/92

-----

update:

now it is possible to introduce a custom delay (use tgcf v0.1.28 or above)
for details read #122 

for full schema of the configuration file [read this](https://github.com/aahnik/tgcf/wiki/How-to-configure-tgcf-%3F#schema)

**Reply by vmussa** [reply]
Hello, @aahnik, first of all, thank you very much for your work and for sharing it with us. The provided link tells us about flood wait errors but doesn't say anything about bans. Do you know if it is possible to be banned from Telegram using `tgcf`?

**Reply by aahnik** [reply]
hi @vmussa , for more info on bans, please read this https://github.com/aahnik/tgcf/discussions/90#discussioncomment-653468 and [this](https://github.com/aahnik/tgcf/wiki/Flood-wait-errors-and-bans)
